assets/css/woocommerce.scss

```diff
@@ -340,37 +340,37 @@ p.demo_store,
 						display: inline-block;
 						padding: 0.5em 0;
 						font-weight: 700;
-						color: $secondarytext;
+						color: var( --wc-secondary )text;
 						text-decoration: none;
 
 						&:hover {
 							text-decoration: none;
-							color: lighten($secondarytext, 10%);
+							color: lighten(var( --wc-secondary )text, 10%);
```

Comment:
Those aren't possible, SCSS needs some value to be able to process using the `lighten()` function, so this will cause a compilation error, also there's a syntax error here, since `var( --wc-secondary )text` isn't valid.

Reply:
Those SCSS functions will need to get removed, at least seems like we are using `lighten()` and `darken()`, I haven't yet checked if anything else needs to be replaced, but this two functions are used in many ways, what makes even harder to try to keep, and feels like will require a small refactoring... At least for what I could remember CSS, feels like that we could use [CSS filters](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) to some, but probably will not work when trying to only apply to a border or something like, and feels like that we could introduce some more variables and start to use `rgba()` or `hsla()` to help defining those other colors.